### ISSUE TITLE
Could not start Flow server due to limited disk space

### ISSUE THREAD

**Issue by XDTZ** [facebook/flow#3438]
I downloaded 0.40.0 version and tried to run it with an empty .flowconfig file. Then I get the following error:
```
Launching Flow server for /mnt/c/projects/jstraining
Spawned flow server (pid=34)
Logs will go to /tmp/flow/zSmntzSczSprojectszSjstraining.log
Error: Failed to start server. exited prematurely with code 2.
Could not start Flow server!
```
My stack trace file was posted here: Microsoft/BashOnWindows#1732
It seems Flow is asking for much(28GB?) shared memory. But I'm running a Surface pro 3 with 4GB RAM and 14.7 GB disk space left. 

Flow runs fine on another machine with much more disk space left and 8GB RAM.

Any help is much appreciated : )

**Comment by gosukiwi** [comment]
Why is this closed? I have the same issue, running version 0.44.1. I'm guessing it's related to #2152 ?

**Comment by XDTZ** [comment]
I did not know other systems were having this issue. I thought this was due to flaws in Bash on Windows. Looks like I should reopen it? Honest I don't have the knowledge to discern the real cause.

**Comment by gosukiwi** [comment]
I don't think this issue should be reopened, it seems like it's a duplicate of #2152 😄
